index.html

```diff
@@ -229,10 +229,10 @@ <h3 class="hero-text hero-text-white">
                 If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
             </h3>
 
-            <form action="" class="contact-form">
-                <input type="text" name="" id="" placeholder="Yeremias" >
-                <input type="email" name="" id="" placeholder="natajayanj14@gmail.com" >
-                <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here" ></textarea>
+            <form action="https://formspree.io/f/xjvdwrqb" method="POST" class="contact-form">
+                <input required type="text" name=""  id="" placeholder="Ingresa tu nombre" maxlength="30" >
+                <input required type="email" name="" id="" placeholder="Ingresa tu correo" >
+                <textarea name="coment" cols="30" rows="10" placeholder="Ingresa tus comentarios" maxlength="500"></textarea>
                 <input class="btn" type="submit" value="Get in touch">
             </form>
```

Comment:
- [ ] Kindy ensure to remove the default input border, on enable and not on focus, either by making the `outline` property zero or by setting the `border` color to transparent. 😃 

- Your design,
<img width="379" alt="wrong" src="https://user-images.githubusercontent.com/41163571/220481443-0b266adc-7e90-43c5-aa54-cc59abb718b4.png">

- Figma design,
<img width="223" alt="right" src="https://user-images.githubusercontent.com/41163571/220481480-1a1b246b-cf2f-4641-b396-b5e1a10a5aae.png">